requirements.txt

```diff
@@ -0,0 +1,135 @@
+ansiwrap==0.8.4
```

Comment:
Should docker be looking for this file? (To auto install all the packages). If so, it didnt grab flask. I tried installing manually, but after doing that flask run doesnt actually do anything